src/components/JdsDataTable/DataTable.vue

```diff
@@ -166,6 +211,12 @@ export default {
       return false
     },
 
+    getColumnLength() {
```

Comment:
**super minor**

should probably use a noun for computed _property_.

Reply:
I keep forgetting about this stuff 😆 , thank you @adrianpdm fixed [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/90/commits/103f5ec205fcb4439fa9366f856fc3b905923908)